backend/src/main/java/co/inventorsoft/scripty/model/entity/MapConverter.java

```diff
@@ -0,0 +1,42 @@
+package co.inventorsoft.scripty.model.entity;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author A1lexen
+ */
+
+@Converter
+public class MapConverter implements AttributeConverter<Map<String, String>, String> {
+    ObjectMapper mapper = new ObjectMapper();
+
+    @Override
+    public String convertToDatabaseColumn(Map<String, String> data) {
+        String value = "";
+        try {
+            value = mapper.writeValueAsString(data);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
```

Comment:
Please, do not use `Throwable#printStackTrace()` method. We have a logger, and all exceptions should be logged using it. Besides, it's poor exception handling. This exception will only occur if we have corrupted data inside database, and we want to know about it. So please, wrap this one in `ApplicationException` and rethrow